helmfile/charts/notify-api/templates/deployment.yaml

```diff
@@ -103,51 +95,51 @@ spec:
                     name: notify-api
                     key: CRM_GITHUB_PERSONAL_ACCESS_TOKEN
             - name: CRM_ORG_LIST_URL
-              value: https://raw.githubusercontent.com/cds-snc/gc-organisations/main/data/all.json
```

Comment:
Hmm I was not aware of that, interesting.. so that is how we feed the organisations list into notify?